README.md

```diff
@@ -383,8 +383,6 @@ ingress:
     ...
 ```
 
-You can add [annotations](https://github.com/gardener/garden-setup/blob/master/components/ingress-controller/chart/templates/00-ingress-controller.yaml#L249) for the ingress controller load balancer service. This can be used for example to deploy an internal load balancer on your cloud provider (see the example for aws above).
```

Comment:
Hmm in my opinion now this is the correct link: https://github.com/gardener/garden-setup/blob/master/components/ingress-controller/deployment.yaml#L72

Reply:
/needs second-opinion

Reply:
Oh, good catch. This link is from the times when we had the nginx chart as a part of garden-setup, now it is checked out during the deployment.

I think it is probably the best idea to keep the sentence (as it is still true) and just remove the link - most people anyway know how annotations work.

What do you think?

Reply:
lgtm

Reply:
Sounds good to me

Reply:
That sounds like a good idea to me. I can make the changes now, if we're all in agreement?